crates/fluvio-cli/src/smartmodule/create.rs

```diff
@@ -34,8 +35,12 @@ impl CreateSmartModuleOpt {
             wasm: SmartModuleWasm::from_binary_payload(buffer),
             ..Default::default()
         };
+        
+        debug!("creating smart-module: {}", self.name);
```

Comment:
can you use structure logging? https://docs.rs/tracing/0.1.35/tracing/macro.debug.html

Reply:
Did you put in lines 38 and 39? Sorry, I thought I did that.

Reply:
something like this:
```
debug!(name = self.name, "creating smart-module");
```

Reply:
Ok I changed the line to that. Also letting you know how it is done in topic/create:

`debug!("creating topic: {} spec: {:#?}", name, topic_spec);`

Reply:
Yup.  There's lots of code using legacy format.